src/GafferUIModule/PathListingWidgetBinding.cpp

```diff
@@ -1188,7 +1188,14 @@ class PathModel : public QAbstractItemModel
 						{
 							// Qt doesn't use exceptions for error handling,
 							// so we must suppress them.
-							IECore::msg( IECore::Msg::Warning, "PathListingWidget", e.what() );
+							cellVariants = CellVariants(
+								GafferUI::PathColumn::CellData(
+									nullptr,  // value
+									new IECore::StringData( "errorSmall.png" ),  // icon
+									nullptr,  // background
+									new IECore::StringData( e.what() )  // toolTip
+								)
+							);
```

Comment:
Thanks for making this change - I definitely think it's the best place to handle the errors in most cases.

Alas, I've just realised that there was a benefit to your custom handling for the `LocationNameColumn` : it still displayed the location name when an error occured computing the attributes. Could you bring back the internal error handling for just that one column type please, in addition to this central handling? Sorry for the backwards and forwards...

Reply:
No worries, the exception to the exception is handled now in 7d28a1deba20258251e32d249a6c79ea77e9312b where I think it's worth logging the error?

Reply:
Sorry, I don't think I explained myself fully enough. I was hoping for the error indicator _and_ the name, like so :

![image](https://user-images.githubusercontent.com/1133871/189326435-8b46b9c6-1ead-4d1b-a0c6-8ee2371cffb7.png)

I think that's what you had in your original version? I feel like I've spent all my just-one-more-change-please tokens on this pr, and I want to get a release out this morning, so I've just pushed a tweak in https://github.com/GafferHQ/gaffer/pull/4820/commits/5aa7ede88de6038e7dd00272e72cff5b85a24ef1.